### ISSUE TITLE
Calling R stuck at openR.sh (macOS)

### ISSUE THREAD

**Issue by songcai** [jalvesaq/Nvim-R#179]
I am using the latest Git version on macOS. If I run R in an external terminal (R_in_buffer=0) with Tmux, when I call R using "\rf", R does not run but a MacVim window is open showing the content of a file called "openR.sh" as follows

#!/bin/sh
tmux -L NvimR -2   new-session -s NvimR-113 'TERM=screen-256color NVIMR_TMPDIR=/var/folders/mh/v1yyk3sx4nn831fqs_wsynch0000gn/T/Nvim-R-scai NVIMR_COMPLDIR=/Users/scai/.cache/Nvim-R NVIMR_ID=pEjeH9znF4YLlwlKuUmT8A NVIMR_SECRET=FERfhKZGarQm2OkxyfDg R_DEFAULT_PACKAGES=datasets,utils,grDevices,graphics,stats,methods,nvimcom R'

If I set R_in_buffer=1, everything is fine.

**Comment by jalvesaq** [comment]
Please, see this: https://github.com/jalvesaq/Nvim-R/issues/177

**Comment by jalvesaq** [comment]
If someone has just bought a new Mac OS X computer and has not changed anything in its configuration yet, what will work, `openR` or `openR.sh`?
Is there a more appropriate way of opening a terminal emulator in Mac OS X via vimscript than writing the command in a shell script and running `open shell_script`?

**Comment by songcai** [comment]
Thanks for the information. 

First, after reverting all "openR.sh" in Nvim-R/R/extern_term.vim back to "openR", everything worked as before.

Second, I am pretty certain that the default shell of macOS terminal is bash. It has been the case and it is still true for the newest version of the macOS. I tried a few old Mac Machines (including mine), all work fine only if I use "openR" (instead of "openR.sh"). I also tried it on an iMac that  was bought late last year, I also have to use "openR". Note that the OS on that iMac has been updated to the newest version and the Terminal and R had never been used on is, so all settings are original.

Third, according to the above trials, I guess the issue #177 is due to coralzhang's personal setting but is not a general issue. It would be great if coral can confirm this.
